packages/amplify-e2e-tests/src/__tests__/__snapshots__/function_9a.test.ts.snap

```diff
@@ -18,8 +18,8 @@ exports.handler = async (event) => {
         statusCode: 200,
     //  Uncomment below to enable CORS requests
     //  headers: {
-    //      \\"Access-Control-Allow-Origin\\": \\"*\\",
-    //      \\"Access-Control-Allow-Headers\\": \\"*\\"
+    //      "Access-Control-Allow-Origin": "*",
+    //      "Access-Control-Allow-Headers": "*"
     //  },
```

Comment:
why do we even keep this?

Reply:
yes, we keep them as suggestions in the lambda code template